src/modules/projectAdministration/components/EditFunderPage.tsx

```diff
@@ -77,29 +108,7 @@ const EditFunderPage = ({ create = false }: { create?: boolean }) => {
       formRole={RecordFormRole.Funder}
       inputVariables={{ projectId }}
       record={funder || undefined}
-      title={
-        !create &&
-        funder && (
-          <ProjectFormTitle
-            title={title}
-            project={project}
-            actions={
-              <DeleteMutationButton<
-                DeleteFunderMutation,
-                DeleteFunderMutationVariables
-              >
-                queryDocument={DeleteFunderDocument}
-                variables={{ input: { id: funder.id } }}
-                idPath={'deleteFunder.funder.id'}
-                recordName='Funder'
-                onSuccess={onSuccessfulDelete}
-              >
-                Delete Record
-              </DeleteMutationButton>
-            }
-          />
-        )
-      }
+      title={titleComponent}
```

Comment:
It seems like we should be able to use the same title component when adding a new funder, the only thing that needs to be hidden is the delete button. This change feels a little simpler, I think?
```suggestion
      title={
        <ProjectFormTitle
          title={title}
          project={project}
          actions={
            funder && (
              <DeleteMutationButton<
                DeleteFunderMutation,
                DeleteFunderMutationVariables
              >
                queryDocument={DeleteFunderDocument}
                variables={{ input: { id: funder.id } }}
                idPath={'deleteFunder.funder.id'}
                recordName='Funder'
                onSuccess={onSuccessfulDelete}
              >
                Delete Record
              </DeleteMutationButton>
            )
          }
        />
      }
```

Reply:
100% prefer. Thank you!